Error: Cannot find module 'xctestwd' when run app-inspector

Environment check:

```bash
$ npm i macaca-cli -g && macaca doctor
```

---

- app inspector version / 版本号:2.0.15


- os version / 系统: Mac OS X 10.14.1 


- device  information / 设备版本:iPhone 6S


- terminal log / 终端输出信息:
>app-inspector -u FGGC0B8E-311A-DF00-8D00-12D781DC222D
{ Error: Cannot find module 'xctestwd'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/app-inspector/lib/ios.js:5:18)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }

> try `app-inspector -u xxxx --verbose` for more log detail.
server.js:16:12 [master] pid:5877 server start with config:
 { port: 5678,
  verbose: true,
  udid: 'FFEC0B8E-501B-4F99-8D38-15D991DB313D',
  ip: '192.168.1.2',
  host: 'MACHOST',
  loaded_time: '2018-11-28 17:32:35' }
>> app-inspector.js:45:12 [master] pid:5877 server start at: http://192.168.1.2:5678
{ Error: Cannot find module 'xctestwd'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/app-inspector/lib/ios.js:5:18)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }

--- a/docs/guide/install.md
+++ b/docs/guide/install.md
@@ -10,7 +10,9 @@ To install app-inspector, [Node.js](https://nodejs.org) environment is required.
 $ npm install macaca-cli -g
 ```
 
-Your environment needs to be setup for the particular mobile platforms that you want to view. Install Android SDK for Android, Xcode for iOS.
+Your environment needs to be setup for the particular mobile platforms that you want to view. 
+
+Refer to [Macaca Environment Setup doc](https://macacajs.github.io/guide/environment-setup.html) to install Android SDK for Android, Xcode for iOS.
 
 Verify the environment with macaca-cli.
 

--- a/docs/zh/guide/install.md
+++ b/docs/zh/guide/install.md
@@ -10,7 +10,10 @@
 $ npm install macaca-cli -g
 ```
 
-你需要准备好你需要进行查看的移动平台的环境。Android 请安装 Android SDK，iOS 安装 Xcode.
+你需要准备好你需要进行查看的移动平台的环境。
+
+请参考[Macaca 环境配置文档](https://macacajs.github.io/zh/guide/environment-setup.html)。Android 请安装 Android SDK，iOS 安装 Xcode。
+
 然后使用 macaca 命令行工具检测环境是否准备好。
 
 ```bash

